playground/locations-site/sites-config/features.json

```diff
@@ -21,13 +21,25 @@
     {
       "$id": "location-stream",
       "filter": {
-        "entityTypes": ["location"]
+        "entityTypes": [
+          "location"
+        ]
       },
-      "fields": ["id", "uid", "meta", "address", "slug"],
+      "fields": [
+        "id",
+        "uid",
+        "meta",
+        "address",
+        "slug"
+      ],
       "localization": {
-        "locales": ["en"],
+        "locales": [
+          "en"
+        ],
         "primary": false
```

Comment:
Not necessarily part of this PR, but I thought we were removing primary from localization?

Reply:
We did, but it doesn't hurt anything and this PR isn't about cleaning up the playgrounds.